dev-lang/jimtcl/files/jimtcl-0.78-bootstrap.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/autosetup/autosetup-find-tclsh b/autosetup/autosetup-find-tclsh
+index dfe70f8..a0f1ae6 100755
+--- a/autosetup/autosetup-find-tclsh
++++ b/autosetup/autosetup-find-tclsh
+@@ -5,10 +5,7 @@
+ d=`dirname "$0"`
+ { "$d/jimsh0" "$d/autosetup-test-tclsh"; } 2>/dev/null && exit 0
+ PATH="$PATH:$d"; export PATH
+-for tclsh in $autosetup_tclsh jimsh tclsh tclsh8.5 tclsh8.6; do
+-	{ $tclsh "$d/autosetup-test-tclsh"; } 2>/dev/null && exit 0
+-done
+-echo 1>&2 "No installed jimsh or tclsh, building local bootstrap jimsh0"
++echo 1>&2 "Bootstrap jimsh0"
+ for cc in ${CC_FOR_BUILD:-cc} gcc; do
+ 	{ $cc -o "$d/jimsh0" "$d/jimsh0.c"; } 2>/dev/null || continue
```

Comment:
Make it respect CFLAGS & co, please. Otherwise adding debug symbols is painful.

Reply:
`jimsh0` is not included into installation tree (a build time requisite).
Could I just add `-ggdb3` in place?

Reply:
I've added `$CFLAGS`.